src/clib/CMakeLists.txt

```diff
@@ -291,6 +291,12 @@ if (ADIOS_BP2NC_TEST)
     PUBLIC _ADIOS_BP2NC_TEST)
 endif ()
 
+if (ADIOS_NO_DECOMPS)
+  message(STATUS "No decomposition data will be saved to ADIOS BP files to reduce disk usage")
+  target_compile_definitions (pioc
+    PUBLIC _ADIOS_NO_DECOMPS)
```

Comment:
Can you add an SPIO prefix to this flag (_SPIO_ADIOS_NO_DECOMPS)? You can add the prefix to other PUBLIC flags as well, if needed.

Reply:
OK, we can add SPIO prefix. For other existing flags, we can change them later in a different PR.